packages/suite/src/utils/wallet/ethUtils.ts

```diff
@@ -24,14 +25,16 @@ export const strip = (str: string): string => {
 };
 
 export const validateAddress = (address: string): string | null => {
-    const hasUpperCase = new RegExp('^(.*[A-Z].*)$');
     if (address.length < 1) {
         return 'Address is not set';
     }
     if (!EthereumjsUtil.isValidAddress(address)) {
         return 'Address is not valid';
     }
-    if (address.match(hasUpperCase) && !EthereumjsUtil.isValidChecksumAddress(address)) {
+    if (
+        address.match(HAS_UPPERCASE_LATER_REGEXP) &&
```

Comment:
you can change `match` to `test`. `match` will return array of matched results or null, `test` is just for quick boolean check

 => https://teamtreehouse.com/community/regular-expressions-test-match-are-these-used-interchangeably

Reply:
maybe now is make sense to use `hasUppercaseLetter` function, I will leave it to you which option is better

Reply:
I think it is safer right now not to change the business logic of it.
It was originally using the `const hasUpperCase = new RegExp('^(.*[A-Z].*)$');` which is the same that is imported from `HAS_UPPERCASE_LATER_REGEXP`. The only think I did is to use the regex definition that now come from the utility package.

https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/utils/wallet/ethUtils.ts#L26

Reply:
ok, resolving